database/src/test/kotlin/no/nav/su/se/bakover/database/RevurderingPostgresRepoTest.kt

```diff
@@ -309,7 +327,7 @@ internal class RevurderingPostgresRepoTest {
             repo.lagre(simulert)
 
             val tilAttestering =
-                simulert.tilAttestering(oppgaveId = OppgaveId("oppgaveId"), saksbehandler = saksbehandler)
+                simulert.tilAttestering(attesteringsoppgaveId = OppgaveId("oppgaveId"), saksbehandler = saksbehandler)
```

Comment:
Kunne kanskje hatt `OppgaveId("attesteringsoppgaveId")` for å kunne skille på dette i assertene.